dirsrvtests/tests/tickets/ticket2058_test.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import pytest
+import os
+import subprocess
+import time
+import ldap
+from lib389._constants import *
```

Comment:
Please, try to avoid mass imports. It makes code less readable and less explicit (which is python coding best practices).

Reply:
@droideck This is probably from the create test template, I think @mreynolds389 has done a cleanup from it recently....